src/table/pagination-table/components/TableWrapper.tsx

```diff
@@ -122,6 +128,7 @@ function TableWrapper(props: TableWrapperProps) {
         tabIndex={-1}
         role="application"
         data-testid="table-container"
+        onScroll={onScroll}
```

Comment:
This is a bit messy because we manually add a listener to this element, in use-scroll-listener. For some reason we are listening to `wheel` events, which is not the same as `scroll`. So this should actually be the better way of doing it, and consider adding listener here as well

Reply:
In other words, ignore this comment